src/main/java/pl/craftserve/radiation/LugolsIodinePotion.java

```diff
@@ -190,18 +189,23 @@ public void onBrew(BrewEvent event) {
         });
     }
 
-    private void convert(PotionMeta potionMeta) {
+    public PotionMeta convert(PotionMeta potionMeta) {
```

Comment:
Can we make it `protected` instead?

Reply:
I'd rather make it public so it can be used by other plugins, eventually public method that creates itemstack using convert